src/main/java/com/bmw/hmm_lib/MostLikelySequence.java

```diff
@@ -54,26 +54,38 @@
      * Since there are no previous states for t=1, backPointerSequence starts with t=2.
      */
     public final List<Map<S, S>> backPointerSequence;
+    private final int brokenTimeStep;
 
-    public MostLikelySequence(List<S> mostLikelySequence, boolean isBroken,
+    public MostLikelySequence(List<S> mostLikelySequence, boolean isBroken, int brokenTimeStep,
             List<Map<S, S>> backPointerSequence, List<Map<S, Double>> messageHistory) {
         this.sequence = mostLikelySequence;
         this.isBroken = isBroken;
+        this.brokenTimeStep = brokenTimeStep;
         this.messageHistory = messageHistory;
         this.backPointerSequence = backPointerSequence;
     }
+    
+    public int getBrokenTimeStep() {
```

Comment:
Please add Javadoc that explains that the returned index is zero-based. Also, please add Junit assertions for this method in [HmmBreakTest.java](https://github.com/bmwcarit/hmm-lib/blob/master/src/test/java/com/bmw/hmm_lib/HmmBreakTest.java).